### PR TITLE
Develop_Moyo_header_Serdiuk_Max

### PR DESCRIPTION
**Pull request by serdiukm**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,62 @@
...
+}
+
+.nav__list {
+  height: 15px;
+  list-style: none;
+  display: flex;
+  margin-right: 60px;
+}
+
+.nav__item {
```
Comment (Viktor-Kost) [inline src/style.css:34]: first item shouldn't have margin
```suggestion
.nav__item:not(:first-child) {
```

File: src/index.html [hunk 2]
```diff
@@ -5,10 +5,47 @@
...
   <body>
     <h1>Moyo header</h1>
+    <header>
+
+      <div class="link__logo"><a href="index.html"><img src="images/logo.png" alt="logo"></a></div>
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a href="index.html" class="is-active">Apple</a>
```
Comment (Viktor-Kost) [inline src/index.html:23]:
```suggestion
            <a href="index.html" class="nav__link is-active">Apple</a>
```

File: src/index.html [hunk 3]
```diff
@@ -5,10 +5,47 @@
...
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="link">Samsung</a>
```
Comment (Viktor-Kost) [inline src/index.html:26]:
```suggestion
            <a href="index.html" class="nav__link">Samsung</a>
```

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,62 @@
...
+  font-size: 12px;
+  color: black;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  background: #00acdc;
```
Comment (Viktor-Kost) [inline src/style.css:51]: move repeated colors to variables

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,62 @@
...
+  border-radius: 2px;
+  bottom: -20px;
+  left: 0;
+  right: 0;
+}
+
+a:hover {
```
Comment (Viktor-Kost) [inline src/style.css:58]: style class selectors instead of tag ones

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,62 @@
...
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  background: #00acdc;
+  border-radius: 2px;
+  bottom: -20px;
```
Comment (Viktor-Kost) [inline src/style.css:53]:
```suggestion
  bottom: 0;
```

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,62 @@
...
+.nav__item {
+  margin-left: 20px;
+}
+
+.link,
+.is-active {
+  text-transform: uppercase;
+  text-decoration: none;
+  font-size: 12px;
+  color: black;
```
Comment (Viktor-Kost) [inline src/style.css:43]: use hex names instead of reserved words to set a color

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Yor demo link is not working, please fix it and make request review again

**Comment by serdiukm** [comment]
> _ No description provided. _

**Pull request closed without merging**